doc/connectivity/functional_connectomes.rst

```diff
@@ -0,0 +1,239 @@
+    :ref:`example_connectivity_plot_signal_extraction.py`
+
+
+.. topic:: **Exercise: computing the correlation matrix of rest fmri**
+   :class: green
+
+   Try using the information above to compute the correlation matrix of
+   the first subject of the ADHD dataset downloaded with
+   :func:`nilearn.datasets.fetch_adhd`
+
+   **Hints:**
+
+   * Inspect the '.keys()' of the object returned by
+     :func:`nilearn.datasets.fetch_adhd`
+
+   * :func:`numpy.corrcoef` can be used to compute a correlation matrix
+     (check the shape of your matrices)
+
+   * :func:`matplotlib.pyplot.imshow` can show a correlation matrix
+
+   * The example above has the solution
+
+|
+
+Time-series from a probabilistic atlas
+========================================
+
+Probabilistic atlases
+----------------------
+
+The definition of regions as by a continuous probability map captures
+better our imperfect knowledge of boundaries in brain images (notably
+because of inter-subject registration errors). One example of such an
+atlas well suited to resting-state data analysis is the `MSDL atlas
+<https://team.inria.fr/parietal/research/spatial_patterns/spatial-patterns-in-resting-state/>`_ (:func:`nilearn.datasets.fetch_msdl_atlas`).
+
+Probabilistic atlases are represented as a set of continuous maps, in a
+4D nifti image. Visualization the atlas thus requires to visualize each
+of these maps, which requires accessing them with
+:func:`nilearn.image.index_img` (see the :ref:`corresponding example <example_manipulating_visualizing_plot_probabilistic_atlas.py>`).
+
+.. image:: ../auto_examples/manipulating_visualizing/images/plot_probabilistic_atlas_1.png
+   :target: ../auto_examples/manipulating_visualizing/plot_probabilistic_atlas.html
+   :scale: 60
+
+
+Extracting signals from a probabilistic atlas
+----------------------------------------------
+
+.. currentmodule:: nilearn.input_data
+
+As with extraction of signals on a parcellation, extracting signals from
+a probabilistic atlas can be done with a "masker" object:  the
+:class:`nilearn.input_data.NiftiMapsMasker`. It is created by
+specifying the important parameters, in particular the atlas::
+
+    from nilearn.input_data import NiftiMapsMasker
+    masker = NiftiMapsMasker(maps_img=atlas_filename, standardize=True)
+
+The `fit_transform` method turns filenames or `NiftiImage objects
+<http://nipy.org/nibabel/nibabel_images.html>`_ to time series::
+
+    time_series = masker.fit_transform(frmi_files, confounds=csv_file)
+
+The procedure is the same as with `brain parcellations
+<parcellation_time_series>`_ but using the :class:`NiftiMapsMasker`, and
+the same considerations on using confounds regressors apply.
+
+.. image:: ../auto_examples/connectivity/images/plot_probabilistic_atlas_extraction_1.png
+   :target: ../auto_examples/connectivity/plot_probabilistic_atlas_extraction.html
+   :scale: 30
+
+
+.. topic:: **Full example**
+
+    A full example of extracting signals on a probabilistic:
```

Comment:
perhaps it's me, but "full example" appear wierd to me, "comprehensive/hands-on example?"